src/tests/roc_audio/test_mixer.cpp

```diff
@@ -20,11 +20,12 @@ namespace audio {
 
 namespace {
 
-enum { BufSz = 100, MaxSz = 500 };
+enum { BufSz = 100, MaxBufDuration = (core::nanoseconds_t) 11345000, SampleRate = 44100,
+       ChannelMask = 0x1, MaxBufSz = 500 };
```

Comment:
OK, so `11345000` was computed as `500 * 1000000000 / 44100`, i.e. `MaxBufSz * core::Second / SampleRate`, right? Can we code it explicitly? Let's compiler to do the job for us. I mean something like:

```
const core::nanoseconds_t MaxBufDuration = MaxBufSz * core::Second / SampleRate;
```

It would be both more readable and easier to maintain (since there will be no need to recompute it if MaxBufSz or SampleRate is changed).

The same for other tests...

Reply:
done.